libgeopmd/src/GPUTopo.cpp

```diff
@@ -41,8 +43,10 @@ namespace geopm
             }
         }
         catch (const Exception &ex) {
+#ifdef GEOPM_DEBUG
```

Comment:
What about here re: verbosity instead of GEOPM_DEBUG?

Reply:
Thanks! Oops